xrpl-std/src/host/host_bindings_for_testing.rs

```diff
@@ -207,7 +209,7 @@ pub unsafe fn compute_sha512_half(
     _out_buff_ptr: *mut u8,
     _out_buff_len: usize,
 ) -> i32 {
-    32
+    -1
```

Comment:
I think this will break some unit tests (can't remember which one). Basically, when a test runs, we want to indicate that 32 bytes were written (even though nothing was actually written) so that the test will move-on and not fail.

(Note: Eventually we'll probably want to implement some of these host functions for real, but the test referenced above doesn't actually use the value from this function ATM so it's fine to sort of mock it out by returning `32` bytes written).

Reply:
I do see this isn't breaking any test _in this build_, so if you want to change this to `-1` for now, it's probably fine. Eventually it will break some unit tests, but if you want we can make that change then?